src/core/qgsrendercontext.cpp

```diff
@@ -38,7 +38,8 @@ QgsRenderContext::QgsRenderContext()
 }
 
 QgsRenderContext::QgsRenderContext( const QgsRenderContext &rh )
-  : mFlags( rh.mFlags )
+  : QgsAbstractTemporal()
```

Comment:
```suggestion
  : QgsAbstractTemporal( rh )
```

Reply:
The operator= method will also need to copy the temporal attributes

Reply:
Can you also expand on the QgsRenderContext copy/assignment unit tests to include checks of the temporal attributes?

Reply:
Right, working on it

Reply:
@nyalldawson why should we pass rendercontext to QgsTemporalRangeObject as in your suggestion?, `QgsTemporalRangeObject( rh )`

Reply:
> @nyalldawson why should we pass rendercontext to QgsTemporalRangeObject as in your suggestion? `QgsAbstractTemporal( rh )` which will now be`QgsTemporalRangeObject( rh )`

Ooh I see @nyalldawson, so we can copy over the temporal attributes...